frontend/src/views/GSecrets.vue

```diff
@@ -589,6 +567,20 @@ export default {
       const tableKeys = mapKeys(sortableTableHeaders, ({ key }) => key)
       return mapValues(tableKeys, () => () => 0)
     },
+    computeItems (secretBinding, relatedShootCount) {
+      return {
+        name: secretBinding.metadata.name,
+        hasOwnSecret: hasOwnSecret(secretBinding),
+        secretNamespace: secretBinding.secretRef.namespace,
+        secretName: secretBinding.secretRef.name,
+        providerType: secretBinding.provider.type,
+        relatedShootCount,
+        relatedShootCountLabel: this.relatedShootCountLabel(relatedShootCount),
+        secretBinding,
+        highlighted: this.highlightedItem === secretBinding.metadata.name,
```

Comment:
highlighted shoot should be as hashparam (instead of subpath). Also remove selection after a few seconds or after click